### PR TITLE
Typeahead v2

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/test-utils.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
 ﻿import { render } from '@testing-library/react';
 import React from 'react';
 import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+<<<<<<< HEAD
```
Comment (atcampbell) [inline src/test-utils.js]: git merge left in

File: src/components/Typeahead.js [hunk 2]
```diff
@@ -69,15 +129,65 @@ function Typeahead({ fetchItems, items, title, loading, clearSearch }) {
...
-            {loading ? <Loading /> : results()}
+            {modal ? (
+                <Dialog
+                    data-testid="modal"
+                    open={dialogOpen}
+                    onClose={() => setDialogOpen(false)}
+                    fullWidth
+                    maxWidth="md"
+                >
+                    <div>
```
Comment (atcampbell) [inline src/components/Typeahead.js]: Fragments instead of these divs?

File: src/components/Typeahead.js [hunk 3]
```diff
@@ -69,15 +129,65 @@ function Typeahead({ fetchItems, items, title, loading, clearSearch }) {
...
+            {!modal ? <Title text={title} /> : <Fragment />}
+            <InputField
+                adornment={
+                    <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        width="24"
+                        height="24"
+                        viewBox="0 0 24 24"
+                    >
+                        <path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z" />
```
Comment (atcampbell) [inline src/components/Typeahead.js]: If this is the same as in the `SearchInputField` the path could be in its own component and reused?

File: src/test-utils.js (hunk 1 above)
Comment (lewisrenfrew) [reply src/test-utils.js]: wow,

**Pull request merged**